modules/cactus-web/src/Header/Header.tsx

```diff
@@ -73,13 +73,31 @@ export const HeaderColumn = styled.div<{ mainColumn?: boolean }>`
   flex-direction: column;
   padding-top: ${(p) => !p.mainColumn && '24px'};
   align-items: center;
+  > div {
+    margin-top: ${(p) => !p.mainColumn && '8px'};
+  }
 
   ${(p) => `
     ${p.theme.mediaQueries?.small}{
+      > div:not(:first-child) {
+        margin-top: ${!p.mainColumn && '0px'};
+      }
+      > div{
+        margin-left: ${!p.mainColumn ? '8px' : '0px'};
+      }
```

Comment:
I think we want to flip the "not first child" selectors.  As it stands, the first child is getting a top margin while the others aren't (see screenshot where the first button is lower than the next two items).  Also, all of the buttons are getting a left margin.  Not sure if that's intended or if you wanted the first item to have no left margin.

```suggestion
      > div {
        margin-top: ${!p.mainColumn && '0px'};
      }
      > div:not(:first-child) {
        margin-left: ${!p.mainColumn ? '8px' : '0px'};
      }
```

![Screen Shot 2021-04-15 at 12 17 35 PM](https://user-images.githubusercontent.com/46460514/114903459-d5ccb880-9de4-11eb-85c8-1f7a06492300.png)

Reply:
@mikesoltow Well, at the first time I omit the left-margin in the first item just to prevent aggregating too much space between the title/mainColumn and the first item